wagtailpolls/forms.py

```diff
@@ -42,7 +43,7 @@ def clean(self):
             self.add_error(None, _('Sorry, we were not able to obtain your ip address'))
         if recent_vote is not None:
             if timezone.now() - recent_vote.time < datetime.timedelta(minutes=cooldown):
-                self.add_error(None, _('Sorry, you cannot vote twice in %(cooldown)s minutes') % {'cooldown': cooldown})
+                self.add_error(None, __('Sorry, you cannot vote twice in %(cooldown)s minutes') % {'cooldown': cooldown})
```

Comment:
This is incorrect. `ungettext` takes three arguments: the singular form, the plural form, and the count. Try:
```python
__('Sorry, you can not vote twice in a minute',
   'Sorry, you can not vote twice in %(cooldown)s minutes',
   cooldown) % {'cooldown': cooldown})
```